dymos/examples/balanced_field/test/test_balanced_field_length.py

```diff
@@ -0,0 +1,452 @@
+import unittest
+
+import numpy as np
+import openmdao.api as om
+from openmdao.utils.testing_utils import use_tempdirs
+from openmdao.utils.assert_utils import assert_near_equal
+import dymos as dm
+from dymos.examples.balanced_field.balanced_field_odes import GroundRollODEComp, TakeoffClimbODEComp
+
+from openmdao.utils.general_utils import set_pyoptsparse_opt
+_, optimizer = set_pyoptsparse_opt('IPOPT', fallback=True)
+
+
+@use_tempdirs
+class TestBalancedFieldLengthRestart(unittest.TestCase):
```

Comment:
These restart cases test that all of the appropriate data is stored in the databases.  Trajectory parameters were previously not being saved before this PR.